fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/EntityModelLayerRegistry.java

```diff
@@ -16,25 +16,18 @@
 
 package net.fabricmc.fabric.api.client.rendereregistry.v1;
 
-import java.util.Objects;
-
-import org.jetbrains.annotations.ApiStatus;
-
 import net.minecraft.client.model.TexturedModelData;
 import net.minecraft.client.render.entity.model.EntityModelLayer;
 
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
-import net.fabricmc.fabric.impl.client.renderer.registry.EntityModelLayerImpl;
-import net.fabricmc.fabric.mixin.client.renderer.registry.EntityModelLayersAccessor;
 
 /**
  * A helpers for registering entity model layers and providers for the layer's textured model data.
  *
- * @deprecated Experimental feature, may be removed or changed without further notice: Snapshot feature.
+ * @deprecated This module has been moved into fabric-rendering-v1. Use {@link net.fabricmc.fabric.api.client.rendering.v1.EntityModelLayerRegistry} instead.
  */
-@Deprecated
-@ApiStatus.Experimental
+@Deprecated(forRemoval = true)
```

Comment:
Not sure about the use of `(forRemoval = true)`, as we don't have plans to remove it for the moment... it should not be a maintenance burden.

Reply:
this should be, as jdk now calls forremoval as terminally deprecated

Reply:
Hmmm, the documentation says that
> `forRemoval=true` indicates that the API is subject to removal in a future release.

So it should be fine to have them.

Reply:
I assumed this was the case since the issue was about reducing the number of modules, and they can't be reduced if it's not removed at some point in the future.